to_fix.txt

```diff
@@ -0,0 +1,120 @@
+./io_scene_nif/modules/nif_import/geometry/vertex/groups.py:                print(f"Vertex {i} has weights not summing to one: {sum_weights[i]}")
```

Comment:
Is this outstanding items?

Reply:
Whoops! That's not supposed to be committed.
I ran a quick grep through all of the repo to find strings I needed to change.
I'll delete this file.